docs/advanced/document-store/qdrant.md

```diff
@@ -61,6 +61,7 @@ da = DocumentArray(
         'host': 'localhost',
         'port': '6333',
         'n_dim': 10,
+        'distance': "cosine",
```

Comment:
Does the user have to specify distance ?

Reply:
okay it's fine let's keep this here, but distance should be removed for the pure filtering example

Reply:
No, users don't need to specify the distance here, I put the distance here just to show that we can support configuring with distance this feature, I can remove it here.